addons/guide_11_multi_account_containers/manifest.json

```diff
@@ -0,0 +1,38 @@
+{
+  "api_version": "0.1",
+  "id": "guide_11_multi_account_containers",
+  "name": "Guide: Multi-Account Containers",
+  "type": "guide",
+  "conditions": {
+    "platforms": ["windows", "macos", "linux"]
+  },
+  "guide": {
+    "id": "guide_multi_account_containers",
+    "title": "Mozilla VPN in the Firefox browser",
+    "advanced": true,
+    "image": "qrc:/addons/guide_11_multi_account_containers/image.svg",
+    "blocks": [
+      {
+        "id": "c_2",
+        "type": "text",
+        "content": "If you use the Multi-Account Containers add-on in Firefox, try adding Mozilla VPN to your unique containers. You can even set different server locations for your unique containers."
```

Comment:
@chrisrooneys 
The repetition of "unique containers" sounds a bit out of place here? For example, the second one could be "for each container".

Reply:
@flodolo Chris is having trouble signing into GitHub from responded on Slack with `I want to keep unique containers for consistency and cognitive load reasons`.